src/xcode/ENA/ENA/Resources/Info_Debug.plist

```diff
@@ -4,8 +4,8 @@
 <dict>
 	<key>BGTaskSchedulerPermittedIdentifiers</key>
 	<array>
-		<string>de.rki.coronawarnapp.exposure-notification</string>
-		<string>de.rki.coronawarnapp.fetch-test-results</string>
+		<string>$(PRODUCT_BUNDLE_IDENTIFIER).exposure-notification</string>
+		<string>$(PRODUCT_BUNDLE_IDENTIFIER).fetch-test-results</string>
```

Comment:
why do you replace it with the environment var?

Reply:
i guess its done to make sure there are no collisions with different app versions, e.g. one that was installed via test flight vs. a debug build? (e.g. the debug target uses de.rki.coronawarnapp-dev, production targets use de.rki.coronawarnapp)

Reply:
It allows us to use the app bundle ID as the prefix for the identifier. In the code, we use Bundle.main.bundleIdentifier as the prefix. This means that we can deploy either app targets during dev (eg from 'de.rki.coronawarnapp' to 'de.rki.coronawarnapp-dev'), and ensure that background tasks from one instance will not collide with the other app's background task identifiers. It's also the method used by Apple's sample code for exposure notifications.